roachtest: gopg failed

[(roachtest).gopg failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2254897&tab=buildLog) on [master@eba9eed5d9a9dabe8a0abed81de78234c133d17d](https://github.com/cockroachdb/cockroach/commits/eba9eed5d9a9dabe8a0abed81de78234c133d17d):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/gopg/run_1
	orm_helpers.go:228,gopg.go:147,gopg.go:160,test_runner.go:754: 
		Tests run on Cockroach v20.2.0-alpha.1-3219-geba9eed5d9
		Tests run against gopg v10.0.1
		191 Total Tests Run
		168 tests passed
		23 tests failed
		0 tests skipped
		7 tests ignored
		0 tests passed unexpectedly
		12 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: v10.TestReadColumnValue - unknown (unexpected)
		--- FAIL: v10.ExampleDB_Model_postgresArrayStructTag - unknown (unexpected)
		--- FAIL: v10.TestConversion - unknown (unexpected)
		--- FAIL: v10.TestGinkgo - unknown (unexpected)
		--- FAIL: v10.TestUnixSocket - unknown (unexpected)
		--- FAIL: v10.TestGocheck - unknown (unexpected)
		--- FAIL: v10.TestBigColumn - unknown (unexpected)
		For a full summary look at the gopg artifacts 
		
		An updated blocklist (gopgBlockList20_2) is available in the artifacts' gopg log

```

<details><summary>More</summary><p>


Artifacts: [/gopg](https://teamcity.cockroachdb.com/viewLog.html?buildId=2254897&tab=artifacts#/gopg)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Agopg.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/gopg.go
+++ b/pkg/cmd/roachtest/gopg.go
@@ -24,6 +24,7 @@ import (
 
 // Currently, we're running a version like 'v9.0.1'.
 var gopgReleaseTagRegex = regexp.MustCompile(`^v(?P<major>\d+)(?:\.(?P<minor>\d+)(?:\.(?P<point>\d+))?)?$`)
+var gopgSupportedTag = "v10.0.1"
 
 // This test runs gopg full test suite against a single cockroach node.
 func registerGopg(r *testRegistry) {
@@ -55,11 +56,12 @@ func registerGopg(r *testRegistry) {
 		}
 
 		t.Status("cloning gopg and installing prerequisites")
-		latestTag, err := repeatGetLatestTag(ctx, c, "go-pg", "pg", gopgReleaseTagRegex)
+		gopgLatestTag, err := repeatGetLatestTag(ctx, c, "go-pg", "pg", gopgReleaseTagRegex)
 		if err != nil {
 			t.Fatal(err)
 		}
-		c.l.Printf("Latest gopg release is %s.", latestTag)
+		c.l.Printf("Latest gopg release is %s.", gopgLatestTag)
+		c.l.Printf("Supported gopg release is %s.", gopgSupportedTag)
 
 		installLatestGolang(ctx, t, c, node)
 
@@ -86,7 +88,7 @@ func registerGopg(r *testRegistry) {
 			c,
 			"https://github.com/go-pg/pg.git",
 			destPath,
-			latestTag,
+			gopgSupportedTag,
 			node,
 		); err != nil {
 			t.Fatal(err)
@@ -124,7 +126,7 @@ func registerGopg(r *testRegistry) {
 		// gopg test suite consists of multiple tests, some of them being a full
 		// test suites in themselves. Those are run with TestGinkgo test harness.
 		// First, we parse the result of running TestGinkgo.
-		if err := gormParseTestGinkgoOutput(
+		if err := gopgParseTestGinkgoOutput(
 			results, rawResults, expectedFailures, ignorelist,
 		); err != nil {
 			t.Fatal(err)
@@ -145,7 +147,7 @@ func registerGopg(r *testRegistry) {
 
 		results.parseJUnitXML(t, expectedFailures, ignorelist, xmlResults)
 		results.summarizeFailed(
-			t, "gopg", blocklistName, expectedFailures, version, latestTag,
+			t, "gopg", blocklistName, expectedFailures, version, gopgSupportedTag,
 			0, /* notRunCount */
 		)
 	}
@@ -162,10 +164,10 @@ func registerGopg(r *testRegistry) {
 	})
 }
 
-// gormParseTestGinkgoOutput parses the summary of failures of running internal
+// gopgParseTestGinkgoOutput parses the summary of failures of running internal
 // test suites from gopg ORM tests. TestGinkgo is a test harness that runs
 // several test suites described by gopg.
-func gormParseTestGinkgoOutput(
+func gopgParseTestGinkgoOutput(
 	r *ormTestsResults, rawResults []byte, expectedFailures, ignorelist blocklist,
 ) (err error) {
 	var (

--- a/pkg/cmd/roachtest/gopg_blocklist.go
+++ b/pkg/cmd/roachtest/gopg_blocklist.go
@@ -39,12 +39,13 @@ var gopgBlockList20_1 = blocklist{
 	"pg | Listener | reconnects on receive error":                     "41522",
 	"pg | Listener | returns an error on timeout":                     "41522",
 	"pg | Listener | supports concurrent Listen and Receive":          "41522",
-	"v9.ExampleDB_Model_postgresArrayStructTag":                       "32552",
-	"v9.TestBigColumn":  "41608",
-	"v9.TestConversion": "32552",
-	"v9.TestGinkgo":     "41522",
-	"v9.TestGocheck":    "17511",
-	"v9.TestUnixSocket": "31113",
+	"v10.ExampleDB_Model_postgresArrayStructTag":                      "32552",
+	"v10.TestBigColumn":       "41608",
+	"v10.TestConversion":      "32552",
+	"v10.TestGinkgo":          "41522",
+	"v10.TestGocheck":         "17511",
+	"v10.TestReadColumnValue": "26925",
+	"v10.TestUnixSocket":      "31113",
 }
 
 var gopgBlockList19_2 = blocklist{
@@ -85,12 +86,13 @@ var gopgBlockList19_2 = blocklist{
 	"pg | soft delete with time column | nil model Deleted allows to select deleted model": "5807",
 	"pg | soft delete with time column | nil model ForceDelete deletes the model":          "5807",
 	"pg | soft delete with time column | nil model soft deletes the model":                 "5807",
-	"v9.ExampleDB_Model_postgresArrayStructTag":                                            "5807",
-	"v9.TestBigColumn":  "5807",
-	"v9.TestConversion": "32552",
-	"v9.TestGinkgo":     "5807",
-	"v9.TestGocheck":    "5807",
-	"v9.TestUnixSocket": "31113",
+	"v10.ExampleDB_Model_postgresArrayStructTag":                                           "32552",
+	"v10.TestBigColumn":       "41608",
+	"v10.TestConversion":      "32552",
+	"v10.TestGinkgo":          "41522",
+	"v10.TestGocheck":         "17511",
+	"v10.TestReadColumnValue": "26925",
+	"v10.TestUnixSocket":      "31113",
 }
 
 var gopgIgnoreList20_2 = gopgIgnoreList20_1

